libs/ui/src/styles.css

```diff
@@ -1,24 +1,14 @@
 @import './tailwind-generated.css';
 
-:root {
-  --gn-font-size-md: 1rem;
-  --gn-font-size-sm: 0.875rem;
-  --gn-title-color: #3B82F6;
-  --gn-link-color: #1D4ED8;
-}
-
 body {
   background-color: var(--color-background);
   color: var(--color-main);
 }
 a {
-  color: var(--gn-link-color);
+  color: var(--color-primary);
```

Comment:
I won't go for that either, what do you think @fxprunayre. Isn't it better to put the `text-primary` class explicitly where we want ?
I'm not sure we want all links to be primary, but maybe CARD metadata title yes, so we can easliy do the green metadata title as in https://forest.eea.europa.eu/datacatalogue

Reply:
> Isn't it better to put the `text-primary` class explicitly where we want ?

It looks like it is the way to go if we want named colors
https://tailwindcss.com/docs/customizing-colors#naming-your-colors